doc/tutorials/content/sources/iccv2011/src/build_all_object_models.cpp

```diff
@@ -90,49 +90,50 @@ main (int argc, char ** argv)
 
   //Parse filter parameters
   std::string filter_parameters_file;
-  pcl::console::parse_argument (argc, argv, "--filter", filter_parameters_file) > 0;
+  if (pcl::console::parse_argument (argc, argv, "--filter", filter_parameters_file) < 0) {
+    pcl::console::print_error ("Missing option --filter\n"); return (1);
```

Comment:
How about adding a function in console.h to do this?
```cpp
void parse_argument_or_exit(argc, argc, name, output) {
  if (parse_argument(argc, argc, name, output) < 0) {
    print_error("Mission option: %s\n", name.c_str());
    return 1;
  }
}
```

Nearly 25 occurrences of this usage across iros and iccv (mainly)

Reply:
The `return 1` won't work. An `exit(1)` could work, but creating a new function just for this (3 or 4 lines) is IMO overkill

Reply:
Yeah, exit 1 would be needed instead of return.

It's a way to keep the changes to a single line instead of looking to save lines by making formatting changes to preserve line numbers. And it's not like it is a one off occurrence.

Reply:
I just checked and it is not necessary to keep the line count the same because the code in those six files is not referenced by line number anywhere. I wrongly assumed it was since in most other tutorials the code in the cpp file is referenced by line numbers from the rst file. So I undid all of my formatting changes and also formatted the added code similar to the surrounding code.
Regarding adding a `parse_argument_or_exit` function: I am still sceptical about this: I don't think it would be used anywhere apart from these two (rather old) tutorials - the behaviour is rather specific, someone might want to exit with a status different than 1, or print a different warning message, or execute some command before exiting, etc.